config/crd/bases/atlas.mongodb.com_atlasprojects.yaml

```diff
@@ -105,15 +105,11 @@ spec:
                         is detected.
                       items:
                         properties:
-                          apiToken:
-                            description: Slack API token or Bot token. Populated for
-                              the SLACK notifications type. If the token later becomes
-                              invalid, Atlas sends an email to the project owner and
-                              eventually removes the token.
-                            type: string
```

Comment:
was not this already removed from secrets removal efforts?

Reply:
Good catch! I went to look into this. In the respective PR, those items were removed from the `_type.go` but the manifests were not (re) generated and the change was missing in the main branch(config/crd/bases). Once they became unexported deserialization was not failing. Gladly, 2.0.0 was not affected because they were generated by the release script and put on the deploy and bundle folders.
cc @s-urbaniak @igor-karpukhin @s-urbaniak